api/valor_api/backend/core/evaluation.py

```diff
@@ -481,11 +449,37 @@ def create_or_get_evaluations(
                 meta={},  # meta stores data about the run after it completes; should be an empty dictionary at creation time
             )
 
+            # unpack filters and params
+            groundtruth_filter = job_request.datum_filter
+            prediction_filter = groundtruth_filter.model_copy()
+            prediction_filter.model_names = [evaluation.model_name]
+            parameters = job_request.parameters
+
+            # verify model and datasets have data for this evaluation
+            if not (
```

Comment:
can we pull these queries out of the if clauses and combine them? including long queries in if statements is hard to read

```
missing_groundtruths = ...
missing_predictions = ...

if missing_groundtruths or missing_predictions:
    evaluation.status = enums.EvaluationStatus.DONE
```